server.js

```diff
@@ -3435,6 +3435,16 @@ mapNugetFeed('myget\\/(.*)', 1, function(match) {
   };
 });
 
+// MyGet (custom tenant, e.g. dotnet.myget.org)
+mapNugetFeed('(.*)\.myget\\/(.*)', 1, function(match) {
```

Comment:
Also, the `\` needs to be escaped, and the 1 turned into a 2 (as there is one more regex grouping).